ui/tests/acceptance/task-logs-test.js

```diff
@@ -75,9 +75,13 @@ module('Acceptance | task logs', function (hooks) {
       run.cancelTimers();
     }, 500);
 
-    await click('button.logs-sidebar-trigger');
+    const taskButton = document.querySelectorAll(
+      'button.logs-sidebar-trigger'
+    )[1];
+    await click(taskButton);
```

Comment:
This test should be using `data-test` attributes to ensure that the correct Task is being selected. However, that's outside of the scope of this PR.

The test, in its previous form was selecting the wrong task which is not a concern of this test:

![image](https://user-images.githubusercontent.com/41024828/194357212-a2f69cab-afb1-4e65-8e56-961140bb2db1.png)